ibazel/BUILD

```diff
@@ -30,29 +34,35 @@ go_binary(
 go_library(
     name = "go_default_library",
     srcs = [
-        "fsnotify.go",
+        "fs_watcher_fsnotify.go",
+        "fs_watcher_darwin.go",
         "ibazel.go",
         "ibazel_unix.go",
         "ibazel_windows.go",
         "lifecycle.go",
         "main.go",
         "main_unix.go",
         "main_windows.go",
-        "source_event_handler.go",
     ],
     importpath = "github.com/bazelbuild/bazel-watcher/ibazel",
     visibility = ["//visibility:private"],
     deps = [
         "//bazel:go_default_library",
         "//ibazel/command:go_default_library",
+        "//ibazel/fsnotify_watcher:go_default_library",
+        "//ibazel/fs_watcher:go_default_library",
         "//ibazel/live_reload:go_default_library",
         "//ibazel/log:go_default_library",
         "//ibazel/output_runner:go_default_library",
         "//ibazel/profiler:go_default_library",
         "//ibazel/workspace_finder:go_default_library",
         "//third_party/bazel/master/src/main/protobuf/blaze_query:go_default_library",
-        "@com_github_fsnotify_fsnotify//:go_default_library",
-    ],
+    ] + select({
+        "@io_bazel_rules_go//go/platform:darwin": [
+            "//ibazel/fsevents_watcher:go_default_library"
```

Comment:
Can you add a comment for future me that describe why this needs to only be included on darwin?